src/node/server/serverPluginCss.ts

```diff
@@ -51,25 +50,22 @@ export const cssPlugin: ServerPlugin = ({ root, app, watcher, resolver }) => {
     }
   })
 
-  watcher.on('change', (file) => {
-    /** filter unused files */
-    if (
-      !Array.from(processedCSS.keys()).some((processed) =>
-        slash(file).includes(processed)
-      ) &&
-      !srcImportMap.has(file)
-    ) {
-      return debugCSS(
-        `${basename(file)} has changed, but it is not currently in use`
-      )
-    }
+  watcher.on('change', (filePath) => {
```

Comment:
I will let the mean of `filePath` and `publicPath` more have clearly and consistence.